index.d.ts

```diff
@@ -174,6 +177,12 @@ declare module "bedrock-protocol" {
     serverId: string
   }
 
+  export interface RealmsOptions {
+    realmId: string
```

Comment:
if these are optional fields they need to be marked as such to avoid type errors